bin/configurator/static/_includes/en/configurator/tab/ci/github-actions/simple/kubernetes-runner/linux/buildah/project.md.liquid

```diff
@@ -28,6 +28,10 @@ This is how the repository that uses werf for build and deploy might look:
 
 Extras:
 
-  * If you do not use ghcr as a container registry, then enter `WERF_REPO`, run [werf cr login]({{ "/docs/reference/cli/werf_cr_login.html" | relative_url }}), and also take into account [features]({{ "/docs/usage/cleanup/cr_cleanup.html#features-of-working-with-different-container-registries" | relative_url }}) of your container registry when cleaning;
+  * If you're not using GitHub Container Registry as your container registry, follow these steps:
+
+    * Set the `WERF_REPO` environment variable to your container registry address;
+    * Log in to the registry using [werf cr login]({{ "/docs/reference/cli/werf_cr_login.html" | relative_url }});
```

Comment:
Mention that this must be done before every command which uses container registry.